openassessment/xblock/reuse_rubric_mixin.py

```diff
@@ -0,0 +1,13 @@
+""" Mixin to encapsulate reuse rubric behavior """
+import logging
+
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+class ReuseRubricMixin:
```

Comment:
I believe this class is unnecessary. You're already creating `is_reuse_rubric_enabled` in `config_mixin` which `OpenAssessment` has access to. I think you can just delete this file and mixin import and everything will still work.